body/stretch_body/stepper.py

```diff
@@ -227,10 +227,6 @@ def set_gains(self,g):
             self.gains=g.copy()
             self._dirty_gains = True
 
-    def write_gains_to_YAML(self):
-        self.params['gains']=self.gains.copy()
-        self.write_device_params(self.name,self.params)
```

Comment:
Although it's unlikely that anyone has used this method besides us, we should keep it and raise a deprecation warning rather than delete it outright.

```
def write_gains_to_YAML(self):
    raise DeprecationWarning('This method has been deprecated since v0.3.0')
```

Deprecated methods can stick around in this way for a few months to give everyone a chance to see it, and then be deleted from the code base.